sinatra-contrib/spec/cookies_spec.rb

```diff
@@ -559,7 +559,7 @@ def cookies(*set_cookies)
     it 'checks response cookies' do
       jar = cookies
       jar['foo'] = 'bar'
-      expect(jar).to include(:foo)
```

Comment:
`Sinatra::Cookies` behaves like a Hash. I'm wondering if there was a change that prevents it from acting like an `IndifferentHash`? If this is just a regular hash, 'foo' and :foo are different keys.

Reply:
I'd say that this spec ensure that the cookie should behave like a indifferent hash, right?
Your change doesn't look like essential solve.

Reply:
The built-in include matcher calls `to_hash` on `Sinatra::Cookies` which returns a regular `Hash`. So to test this it would have to either be:

`expect(jar.include?(:foo)).to be true`

or 

`expect(jar).to include('foo')`

Reply:
You needed to investigate the problem more deeply.

Please take a look at [the rspec issue](https://github.com/rspec/rspec-expectations/issues/1068).
The backword-compatibility breaking isn't intentional, and the original issue was closed by [this patch](https://github.com/rspec/rspec-expectations/issues/1068).
However, our problem hasn't been fixed by that because `Sinatra::Cookies::Jar` doesn't inherit `Hash`. 

The patch seems not to be enough for fixing the breaking.
I think this is the rspec issue, so once I'll try to fix our failure spec by changing our spec.